conans/client/rest/rest_client_v1.py

```diff
@@ -81,18 +82,25 @@ def get_conan_manifest(self, ref):
         contents = {key: decode_text(value) for key, value in dict(contents).items()}
         return FileTreeManifest.loads(contents[CONAN_MANIFEST])
 
-    def get_package_manifest(self, package_reference):
+    def get_package_manifest(self, pref):
         """Gets a FileTreeManifest from a package"""
 
         # Obtain the URLs
-        url = self.conans_router.package_manifest(package_reference)
+        url = self.conans_router.package_manifest(pref)
         urls = self._get_file_to_url_dict(url)
 
         # Get the digest
         contents = self._download_files(urls, quiet=True)
-        # Unroll generator and decode shas (plain text)
-        contents = {key: decode_text(value) for key, value in dict(contents).items()}
-        return FileTreeManifest.loads(contents[CONAN_MANIFEST])
+        try:
+            # Unroll generator and decode shas (plain text)
+            content = dict(contents)[CONAN_MANIFEST]
+            return FileTreeManifest.loads(decode_text(content))
+        except Exception as e:
+            msg = "Error retrieving manifest file for package " \
+                  "'{}' from remote ({}): '{}'".format(pref.full_repr(), self.remote_url, e)
+            logger.error(msg)
```

Comment:
print as error and raise an exception with same message. Is there any chance to print twice on terminal?

Reply:
Yes, once it is raised, it would be enough to log the traceback

Reply:
When we are logging to a file (`CONAN_LOGGING_FILE`) I would want to see printed there the message and the traceback. If I only log the traceback I won't get the `Error retrieving manifest file...` message (unless someone else is logging the exception, and I cannot be sure about that)